### PR TITLE
Add <Input> Component

### PR DESCRIPTION
**Pull request by matthieuh**
Screen Shot 2019-12-05 at 11 05 02

### CODE REVIEW

File: src/components/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+          '&:disabled': {
+            borderColor: 'gray',
+            color: 'gray',
+            cursor: 'not-allowed',
+          },
+          '&:focus': {
+            border: '2px solid',
+            borderColor: 'black',
+            cursor: 'text',
+          },
```
Comment (paulocfjunior) [inline src/components/Input.tsx]: I think we should also have a hover effect here, which (according to the current design on Traefik) will be very similar to this, but with 1px border.

Comment (matthieuh) [reply src/components/Input.tsx]: @paulocfjunior Done 👍

**Comment by matthieuh** [comment]
@paulocfjunior is this PR ready to merge or do you have any other feedback?

**Comment by paulocfjunior** [comment]
I think it's good to go :+1:

**Pull request merged**